sdk/src/impl/position-impl.ts

```diff
@@ -189,6 +190,75 @@ export class PositionImpl implements Position {
     return txBuilder;
   }
 
+  async collectFees(
+    resolveATA: boolean = true,
+    destinationWallet?: Address,
+    positionWallet?: Address,
+    ataPayer?: Address
+  ): Promise<TransactionBuilder> {
+    const [destinationWalletKey, positionWalletKey, ataPayerKey] = AddressUtil.toPubKeys([
+      destinationWallet ?? this.ctx.wallet.publicKey,
+      positionWallet ?? this.ctx.wallet.publicKey,
+      ataPayer ?? this.ctx.wallet.publicKey,
+    ]);
+
+    const whirlpool = await this.fetcher.getPool(this.data.whirlpool, true);
```

Comment:
i think refresh should be false here. We only care about the tokenMint, which will never change. If the cache already has a hit on this whirlpool account, we should leverage it.